tracer/src/Datadog.Trace/Configuration/ImmutableTracerSettings.cs

```diff
@@ -121,6 +121,7 @@ internal ImmutableTracerSettings(TracerSettings settings, bool unusedParamNotToU
             RouteTemplateResourceNamesEnabled = settings.RouteTemplateResourceNamesEnabled;
             DelayWcfInstrumentationEnabled = settings.DelayWcfInstrumentationEnabled;
             WcfObfuscationEnabled = settings.WcfObfuscationEnabled;
+            PropagationStyle = settings.PropagationStyle;
```

Comment:
I don't think we need this? When a user sers `DD_PROPAGATION_STYLE` it means use that value for both `DD_PROPAGATION_STYLE_INJECT` and `DD_PROPAGATION_STYLE_EXTRACT` (unless they are also specified).

Reply:
Oh yeah that makes sense - will remove thanks!